eng/common/scripts/job-matrix/job-matrix-functions.ps1

```diff
@@ -422,7 +429,11 @@ function ProcessImport([MatrixParameter[]]$matrix, [String]$selection, [Array]$n
         }
     }
     if ((!$matrix -and !$importPath) -or !$importPath) {
-        return $matrix, @(), $displayNamesLookup
+        return [PSCustomObject]@{
```

Comment:
Is the cast to PSCustomObject necessary over just a hash?

Reply:
I prefer PSCustomObject for structured data that has the expectation of showing up on the other side reliably. Discussed offline and we'll just leave this alone.